recipes/natsort/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "natsort" %}
+{% set version = "4.0.4" %}
```

Comment:
Hi, my problem is that I want to package a software that depends on this particular version.